management/pre_firebase.js

```diff
@@ -0,0 +1,76 @@
+const requestPromise = require('request-promise');
+
+// [START list_available_projects]
+async function listProjects() {
+  const accessToken = await getAccessToken();
+  const options = {
+    uri: 'https://firebase.googleapis.com/v1beta1/availableProjects',
+    method: 'GET',
+    headers: {
+      'Authorization': 'Bearer ' + '<OAUTH2_ACCESS_TOKEN>',
+    },
+    json: true,
+  };
+
+  try {
+    const resp = await requestPromise(options);
+    console.log(resp);
+    const projects = resp['projectInfo'];
+    console.log('Project total: ' + projects.length);
+    console.log('');
+    for (let i in projects) {
+      const project = projects[i];
+      console.log('Project ' + i);
```

Comment:
Nit: when doing something like this you could do:
```js
console.log(`Project ${i}
ID: ${project['project']}
Display Name: ${project['displayName']
`);
```

Reply:
Done